app/processing/download-and-parse.js

```diff
@@ -0,0 +1,20 @@
+const storage = require('../storage')
+const parsePaymentFile = require('./parse-payment-file')
+const batch = require('./batch')
```

Comment:
similar for this `batch`config-y deal, some places its `batch` others `batches`

Reply:
I've changed all to `batch` as I feel that makes more contextual sense.